packages/react-renderer-demo/src/app/pages/renderer/form-template.md

```diff
@@ -0,0 +1,70 @@
+import Grid from '@material-ui/core/Grid'
+import RawComponent from '@docs/raw-component';
+import RouterLink from 'next/link';
+import Link from '@material-ui/core/Link';
+
+import ListOfContents from '../../src/helpers/list-of-contents';
+
+<Grid container item>
+<Grid item xs={12} md={10}>
+
+# FormTemplate
+
+FormTemplate component allows you to fully customize the form. FormTemplate receives only two props:
```

Comment:
I would probably add the `layout` word there

FormTemplate component allows you to fully customize the form **layout**. FormTemplate receives only two props